docs/example_usage.md

```
@@ -112,14 +112,14 @@ used in the Package Configs to specify the Docker image to use for building the
 Dockerfile must be named `Dockerfile`.
 
 ??? example "Dockerfile example"
-    The following Dockerfile is used for building `curl` and `zlib` Packages for Fedora 41. It
+    The following Dockerfile is used for building `curl` and `zlib` Packages for Fedora 42. It
     installs all required tools and fulfills all
     [requirements](https://github.com/bacpack-system/packager/blob/master/doc/DockerContainerRequiremetns.md).
-    The path to this Dockerfile is `context/docker/fedora41/Dockerfile`. The `fedora41` is the
+    The path to this Dockerfile is `context/docker/fedora42/Dockerfile`. The `fedora42` is the
     name of the Docker image and is used in the Package Configs.
 
     ```docker
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)